assets/locales/en.json

```
@@ -41,7 +35,7 @@
   "restartIpfsDesktop": "Restart IPFS Desktop",
   "openLogs": "Open logs",
   "takeScreenshot": "Take Screenshot",
-  "downloadHash": "Download Hash",
+  "downloadHash": "Download CID",
```

```suggestion
  "downloadHash": "Download from CID...",
```
I think we need the "from" so it's explicit that the user isn't downloading a CID. The ellipses are there to indicate that something will pop up in the next step.

---

I'm just afraid this would get very long. However, let's also note we can use IPNS or IPFS paths and not only a raw CID. I think we need the most generic naming here, but I'm not sure what it might be.

---

What about "just" "Download..."?

---

I think this’ll work for now. Worst case is the user gets to the dialog, decides “that’s not what I meant” and cancels.

---

I'll change then. If we end up finding a better message, we can change it anytime 😃